README.md

```diff
@@ -2,7 +2,7 @@
 
 ---
 
-[![Build Status](https://travis-ci.org/elixir-ecto/ecto.svg?branch=master)](https://travis-ci.org/elixir-ecto/ecto)
+[![Build Status](https://github.com/elixir-ecto/ecto_sql/workflows/CI/badge.svg)](https://github.com/elixir-ecto/ecto/actions)
```

Comment:
```suggestion
[![Build Status](https://github.com/elixir-ecto/ecto/workflows/CI/badge.svg)](https://github.com/elixir-ecto/ecto/actions)
```
The badge is pointing to the right repo?

Reply:
oops, thanks!